data_api/staging/models.py

```diff
@@ -9,6 +9,7 @@
 from django.contrib.postgres.fields import ArrayField, JSONField
 from django.core.exceptions import FieldDoesNotExist
 from django.db import models, transaction
+from django.utils import six
```

Comment:
what's the difference between django's `six` and this one: https://six.readthedocs.io/?

Reply:
is a customized version of the six module.. I don't think is a big difference https://docs.djangoproject.com/en/1.11/topics/python3/